.github/workflows/build_release.yaml

```diff
@@ -55,3 +55,44 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+  build_agent:
+    name: 'Build Agent'
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'openwallet-foundation'
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.inputs.tag || github.ref }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Prepare docker tags for image
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/${{ github.repository }}/agent
+          tags: |
+            type=raw,value=${{ github.event.inputs.tag }}
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=ref,event=tag
```

Comment:
This will tag the agent image with the same version as the endorser image (e.g.: `1.1.1`): would it make more sense to re-tag the agent with the same version as the ACA-Py base image being used? In this case we may need to grab the version from within the Dockerfile with a script sincce there is no input for it.

Thoughts @PatStLouis @i5okie

Reply:
Had a chat with @PatStLouis , we'll tag the agent image with the same version as the API since they will need to be matched to ensure no issues happen.